docs/guides/admin/admin-cli.md

```diff
@@ -570,3 +570,45 @@ On instance['context_link']:
 => OK: "mseventid"
 => OK: "urlhaus"
 ```
+
+### Analyzer-stats
+
+`tsctl`offers a method called `analyzer-stats` to display various information about analyzer runs of the past.
+
+To use the `analyzer-stats` subcommand, you would run `tsctl analyzer-stats` followed by the desired options and arguments.
+
+```shell
+tsctl analyzer-stats
+```
+
+Example:
+
+```shell
+tsctl analyzer-stats --help
+Usage: tsctl analyzer-stats [OPTIONS] ANALYZER_NAME
+
+  Prints analyzer stats.
+
+Options:
+  --timeline_id TEXT         Timeline ID if the analyzer results should be
+                             filtered by timeline.
+  --focus TEXT               Focus on: [many_hits, long_runtime, recent]
```

Comment:
"Focus" is not very intuitive naming IMO. What about `--scope` instead?

Reply:
good point, changed it